app/komponenter/veilederpanel/veilederpanel.module.css

```diff
@@ -8,15 +8,15 @@
   }
 }
```

Comment:
Kan denne klassen brukes uavhengig av poster eller ikke? 

For å sjekke dette må man se på hva padding: 0 egt. gjør av forskjell. 

Burde også bruke variabelen for max-width i stedet for å skrive `768px`

Reply:
Dette kan gjøres i egen PR

Reply:
Eget kort: https://trello.com/c/3zNrAjgj/134-dobbelsjekke-css